.gitignore

```diff
@@ -5,3 +5,4 @@ test/data/bigrams.tsv
 docs/build/
 docs/site/
 .DS_Store
+Manifest.toml
```

Comment:
Do we need this change in this PR? I would focus on adding Tables.jl support only here.

Reply:
This was just to help spread the fun at the JuliaCon hackathon; I'll make sure to remove in the final commit.